src/CMakeLists.txt

```diff
@@ -330,6 +332,13 @@ set (PY_SUPPORT_FILES
 )
 
 #
+# Add the pycapnp Python package to includes
```

Comment:
https://github.com/jparyani/pycapnp/issues/53 With the statically linked GCC we are now using, we may need to take the reins on this issue and help make a pycapnp wheel. No chance via MS compilers, but should be possible via GCC (investigation and wheel building required).